app/routes/toezicht/inzendingen.js

```diff
@@ -12,19 +12,21 @@ export default Route.extend(DataTableRouteMixin, {
     classificatieId: { refreshModel: true },
     provincieId: { refreshModel: true },
     besluitTypeId: { refreshModel: true },
+    toezichtRegulationTypeId: { refreshModel: true },
```

Comment:
I would use `regulationTypeId` to be in line with the property of inzending the filter applies on